.test-infra/jenkins/job_InferenceBenchmarkTests_Python.groovy

```diff
@@ -0,0 +1,170 @@
+        requirements_file     : 'apache_beam/ml/inference/torch_tests_requirements.txt',
+        experiments           : 'no_use_multiple_sdk_containers',
+        publish_to_big_query  : true,
+        metrics_dataset       : 'beam_run_inference',
+        metrics_table         : 'torch_inference_imagenet_results_resnet101',
+        input_options         : '{}', // this option is not required for RunInference tests.
+        influx_measurement    : 'torch_inference_imagenet_resnet101',
+        influx_db_name        : InfluxDBCredentialsHelper.InfluxDBDatabaseName,
+        influx_hostname       : InfluxDBCredentialsHelper.InfluxDBHostUrl,
+        // args defined in the performance test
+        pretrained_model_name : 'resnet101',
+        // args defined in the example.
+        input                 : 'gs://apache-beam-ml/testing/inputs/openimage_50k_benchmark.txt',
+        // TODO: make sure the model_state_dict_path weights are accurate.
+        model_state_dict_path : 'gs://apache-beam-ml/models/torchvision.models.resnet101.pth',
+        output                : 'gs://temp-storage-for-end-to-end-tests/torch/result_' + now + '.txt'
+      ]
+    ],
+    [
+      title             : 'Pytorch Imagenet Classification with Resnet 152',
+      test              : 'apache_beam.testing.benchmarks.inference.pytorch_image_classification_benchmarks',
+      runner            : CommonTestProperties.Runner.DATAFLOW,
+      pipelineOptions: [
+        job_name              : 'benchmark-tests-pytorch-imagenet-python' + now,
+        project               : 'apache-beam-testing',
+        region                : 'us-central1',
+        staging_location      : 'gs://temp-storage-for-perf-tests/loadtests',
+        temp_location         : 'gs://temp-storage-for-perf-tests/loadtests',
+        requirements_file     : 'apache_beam/ml/inference/torch_tests_requirements.txt',
+        experiments           : 'no_use_multiple_sdk_containers',
+        publish_to_big_query  : true,
+        metrics_dataset       : 'beam_run_inference',
+        metrics_table         : 'torch_inference_imagenet_results_resnet152',
+        input_options         : '{}', // this option is not required for RunInference tests.
+        influx_measurement    : 'torch_inference_imagenet_resnet152',
+        influx_db_name        : InfluxDBCredentialsHelper.InfluxDBDatabaseName,
+        influx_hostname       : InfluxDBCredentialsHelper.InfluxDBHostUrl,
+        // args defined in the performance test
+        pretrained_model_name : 'resnet152',
+        // args defined in the example.
+        input                 : 'gs://apache-beam-ml/testing/inputs/openimage_50k_benchmark.txt',
+        // TODO: make sure the model_state_dict_path weights are accurate.
+        model_state_dict_path : 'gs://apache-beam-ml/models/torchvision.models.resnet152.pth',
+        output                : 'gs://temp-storage-for-end-to-end-tests/torch/result_' + now + '.txt'
```

Comment:
will this output conflict with the output from the other tests since it has the same name?

Reply:
Changed it. thanks